WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift

```diff
@@ -43,10 +43,9 @@ final class TotalsViewModelTests: XCTestCase {
 
         await sut.syncOrder(for: [CartItem(id: UUID(), item: item, quantity: 1)], allItems: [item])
         XCTAssertNotNil(sut.order)
-        let order: Order = orderService.order(from: sut.order!)
 
         // When
-        _ = try await cardPresentPaymentService.collectPayment(for: order, using: .bluetooth)
+        _ = try await cardPresentPaymentService.collectPayment(for: sut.order!, using: .bluetooth)
```

Comment:
In here we could avoid the force-unwrapping by failing the test if the order happens to be `nil`:
```suggestion
        guard let order = sut.order else {
            return XCTFail("Expected order. Got nothing")
        }
        _ = try await cardPresentPaymentService.collectPayment(for: order, using: .bluetooth)
```

Reply:
If the order is nil the test will fail in the line above:
`XCTAssertNotNil(sut.order)`

Reply:
But I can change it to this

Reply:
Whatever you prefer is fine, def not a blocker 👍 , I generally just try to avoid force-unwrapping as while the case might be covered now, if we happen to change it in the future without realizing there's a force-unwrap we might introduce a potential crash.